node/src/actor_system_factory.rs

```diff
@@ -1776,8 +1776,10 @@ mod tests {
         let alias_cryptde_public_key_before = public_key_for_dyn_cryptde_being_null(alias_cryptde);
         let actor_factory = Box::new(ActorFactoryReal {}) as Box<dyn ActorFactory>;
         let actor_factory_before_raw_address = addr_of!(*actor_factory);
-        let mut persistent_config =
-            Box::new(PersistentConfigurationMock::default().set_arbitrary_id_stamp(4578112));
+        let persistent_config = Box::new(
+            PersistentConfigurationMock::default()
+                .set_arbitrary_id_stamp(ArbitraryIdStamp::new(457812)),
```

Comment:
It might make sense to have ArbitraryIdStamp be opaque. You'd create one with ::new(), and you'd have no idea what value it contained; the value could be randomized or sequential or calculated from a timestamp or whatever--just not supplied by the user.  It'd be Copy, so you'd hand it into .set_arbitrary_id_stamp() and retain it, and then assert it later.